README.md

```diff
@@ -132,11 +132,11 @@ security group.
 
 Configures environment variables common to *all* environments.
 
-### var_*environment*
+### var_*environment*.env
```

Comment:
Are these just correcting inaccurate docs? It doesn't represent a change — is that right?

Reply:
Yep! The framework does not communicate the error well if you get a config filename wrong, so I thought it best to be explicit - particularly given that `config/var_app` specifically must not end in `.env`